packages/server/customer-os-api/repository/contract_repository.go

```diff
@@ -0,0 +1,98 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j/dbtype"
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-api/entity"
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-api/tracing"
+	"github.com/openline-ai/openline-customer-os/packages/server/customer-os-common-module/utils"
+	"github.com/opentracing/opentracing-go"
+)
+
+type ContractRepository interface {
+	CreateContract(ctx context.Context, tenant, organization string, entity entity.ContractEntity) (*dbtype.Node, error)
+	SetContractCreator(ctx context.Context, tenant, userId, contractId string) error
+}
+
+type contractRepository struct {
+	driver *neo4j.DriverWithContext
+}
+
+func NewContractRepository(driver *neo4j.DriverWithContext) ContractRepository {
+	return &contractRepository{
+		driver: driver,
+	}
+}
+
+func (r *contractRepository) CreateContract(ctx context.Context, tenant, organizationId string, entity entity.ContractEntity) (*dbtype.Node, error) {
```

Comment:
Contract creation is done via events, 
need to call corresponding event via grpc